tests/ops/api/v1/endpoints/test_connection_config_endpoints.py

```diff
@@ -225,7 +225,7 @@ def test_patch_connection_saas_with_secrets_new(
                     },
                     "name": "My Mailchimp Test",
                     "description": "Mailchimp ConnectionConfig description",
-                    "saas_connection_type": "mailchimp",
+                    "saas_connector_type": "mailchimp",
```

Comment:
I found this while doing some searching through the codebase. Technically this API expects `saas_connector_type` so I think it was getting a bad request error, but since this test also *expects* an error, it passes. It was testing the wrong thing though 😉